src/components/MatrixBlock/index.js

```diff
@@ -0,0 +1,19 @@
+import "./MatrixBlok.scss"
+
+export const Matrix = ({ arr }) => {
+	return (
+		<div className="app__matrix-block">
+			<button className="app__add">Add row</button>
+			<div className="app__matrix-body d-flex justify-center align-center">
+				{
```

Comment:
what the code style is. awesome!!! please don't move any code to a new line after the curly bracket